### DISCUSSION TITLE
undefined array key App\Models when trying to do process like CRUD. Authentication is working.

### DISCUSSION THREAD

**Discussion by firdausdsmr** [spatie/laravel-multitenancy#477] [Q&A]
Hi guys, 

can anyone help me with this error. i've wasted so much time but couldnt solve this problem yet. im not sure what this error and where is it coming from. or why im getting this error

<img width="295" alt="image" src="https://github.com/spatie/laravel-multitenancy/assets/107985127/295e6e08-84cb-45ce-a86b-a835076dcc55">

**Comment by masterix21** [comment]
Sorry @firdausdsmr, but the error seems to be unrelated to our package. We can't help you.
